srcpkgs/basu/patches/uid-nobody.patch

```diff
@@ -0,0 +1,39 @@
+--- meson.build
++++ meson.build
+@@ -189,21 +189,21 @@ conf.set('GPERF_LEN_TYPE', gperf_len_type,
+ nobody_user = get_option('nobody-user')
```

Comment:
Not sure this part of the patch is necessary, I'd be okay with still printing the warning, since otherwise it's more stuff to rebase if they change the build system.

Reply:
Fixed.